packages/material/src/controls/MaterialTimeControl.tsx

```diff
@@ -98,16 +98,17 @@ export const MaterialTimeControl = (props: ControlProps) => {
         <TimePicker
           label={label}
           value={value}
-          clearable
           onChange={onChange}
           inputFormat={format}
           disableMaskedInput
           ampm={!!appliedUiSchemaOptions.ampm}
           views={views}
           disabled={!enabled}
-          cancelText={appliedUiSchemaOptions.cancelLabel}
-          clearText={appliedUiSchemaOptions.clearLabel}
-          okText={appliedUiSchemaOptions.okLabel}
+          componentsProps={{
+            actionBar: {
+              actions: (variant) => (variant === 'desktop' ? [] : ['clear', 'cancel', 'accept'])
+            }
+          }}
```

Comment:
Why not use the `cancelLabel`, `clearLabel` etc.?

Reply:
Because this actionBar configuration does not allow to specify labels for the buttons. The values used in `['clear', 'cancel', 'accept']` are enum values that specify which buttons to show.